hudi-common/src/main/java/org/apache/hudi/common/fs/FSUtils.java

```diff
@@ -77,9 +77,10 @@
 public class FSUtils {
 
   private static final Logger LOG = LogManager.getLogger(FSUtils.class);
-  // Log files are of this pattern - .b5068208-e1a4-11e6-bf01-fe55135034f3_20170101134598.log.1
+  // Log files are of this pattern - .b5068208-e1a4-11e6-bf01-fe55135034f3_20170101134598.log.1_1-0-1
+  // Archive log files are of this pattern - .commits_.archive.1_1-0-1
   private static final Pattern LOG_FILE_PATTERN =
-      Pattern.compile("\\.(.*)_(.*)\\.(.*)\\.([0-9]*)(_(([0-9]*)-([0-9]*)-([0-9]*)(-cdc)?))?");
+      Pattern.compile("\\.(.+)_(.*)\\.(.+)\\.(\\d+)(_((\\d+)-(\\d+)-(\\d+))(-cdc)?)?");
```

Comment:
thought we want to use `.cdc` instead of the unconventional `-cdc`?

Reply:
yep. Discussed with @danny0405 , this pr just changes the origin pattern. when this pr is merged, i will change the cdc format in https://github.com/apache/hudi/pull/7042.